Engine/Component/ComponentCamera.cpp

```diff
@@ -240,16 +241,18 @@ void ComponentCamera::RecordFrame(float width, float height)
 
 	glBindFramebuffer(GL_FRAMEBUFFER, 0);
 #endif
+
+	RecordDebugDraws(width, height);
 	
 }
 
-void ComponentCamera::RecordDebugDraws(float width, float height) const
+void ComponentCamera::RecordDebugDraws(float width, float height)
```

Comment:
Why now is not const?

Reply:
Because I'm passing `this` as a parameter within the function to render debug draws and it doesn't allow const